tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -46,27 +50,43 @@ def parameters(cls):
                 value_type=int, min_value=1, optional=True, default=1,
                 description="The number of classes with the highest probability, which will be used to decide "
                             "if prediction is correct."
-            )
+            ),
+            'match': BoolField(optional=True, default=False)
         })
 
         return parameters
 
     def configure(self):
         self.top_k = self.get_value_from_config('top_k')
+        self.match = self.get_value_from_config('match')
 
         def loss(annotation_label, prediction_top_k_labels):
             return int(annotation_label in prediction_top_k_labels)
 
-        self.accuracy = AverageMeter(loss)
+        if not self.match:
+            self.accuracy = AverageMeter(loss)
+        else:
+            self.accuracy = []
 
     def update(self, annotation, prediction):
-        return self.accuracy.update(annotation.label, prediction.top_k(self.top_k))
+        if not self.match:
+            return self.accuracy.update(annotation.label, prediction.top_k(self.top_k))
+        else:
+            accuracy = accuracy_score(annotation.label, prediction.label)
+            self.accuracy.append(accuracy)
```

Comment:
we need to return metric for single sample in quantization case
```suggestion
            self.accuracy.append(accuracy)
            return accuracy
```